Dockerfile.concourse

```diff
@@ -3,10 +3,8 @@ FROM python:3.9
 RUN mkdir test_data
 COPY test_data test_data/
 
-COPY dist/ff_fasttext_api-0.1.0-py3-none-any.whl .
-
-RUN pip install ff_fasttext_api-0.1.0-py3-none-any.whl
+RUN pip install category_api-0.1.0-py3-none-any.whl
```

Comment:
this is installing a very specific version - this should be configured somewhere I would've thought.